aws/cloudformation/s3_backed_subdomain.yml

```diff
@@ -42,6 +42,16 @@ Resources:
     Properties:
       AccessControl: PublicRead
       BucketName: !Ref BucketName
+      CorsConfiguration:
+        CorsRules:
+          - AllowedMethods: [GET]
+            AllowedOrigins:
+              [
+                '*.code.org',
+                '*.code.org:3000',
+                'http://localhost-studio.code.org:*',
```

Comment:
I don't love this, but also in the [AllowedOrigin docs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ManageCorsUsing.html#cors-allowed-origin) -- you can only have 1 wildcard per origin. 

Maybe the origin on line 51 is useless? I just didn't want to assume that everyone uses port 3000 for localhost, although it is our default. This is necessary because `http://localhost-studio.code.org:3000` unfortunately doesn't match `*.code.org`.

I'm definitely open to other approaches here, but I added these as a starting point for allowing resources on localhost.

Reply:
I would be surprised if this is the first place we're making that assumption. You could preemptively add 3001 and 3002 just in case, but I doubt it will be an issue.